### ISSUE TITLE
[LINT_BUG]: Error on non-existing box module

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#108]
### box.linters version

0.9.1

### Sample source code to lint

```R
box::use(path/some_module, )
```

### Lint command used
```R
lintr::lint(
  code,
  linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
)
```

### Lint result

```
Error: Linter 'box_unused_attached_mod_linter' failed in xxxxx: unable to load module “path/some_module”; not found in...
```

### Expected result

Should return lint that module does not exist.

**Comment by SaintRod** [comment]
I'm receiving this error in version 0.10.5 when the module does exist and there is no error in `box::use()`.

example project structure:
- root
  - functions
    - foo
      - a.R
    - bar
      - b.R

In file ./functions/foo/a.R

```
box::use(
  .. / bar / b[some_function]
)
```

There is no error in importing the module when the box code is executed, however, linting file a.R fails with "... unable to load module ..."

```
box::use(
  box.linters[box_default_linters],
  here[here],
  lintr[lint, lint_dir],
)
lint(
  filename = here("functions", "foo", "a.R"),
  linters = box_default_linters$box_mod_fun_exists_linter
)
```

**Comment by radbasa** [comment]
> I'm receiving this error in version 0.10.5 when the module does exist and there is no error in `box::use()`.
> 
> example project structure:
> 
> * root
>   
>   * functions
>     
>     * foo
>       
>       * a.R
>     * bar
>       
>       * b.R
> 
> In file ./functions/foo/a.R
> 
> ```
> box::use(
>   .. / bar / b[some_function]
> )
> ```
> 
> There is no error in importing the module when the box code is executed, however, linting file a.R fails with "... unable to load module ..."
> 
> ```
> box::use(
>   box.linters[box_default_linters],
>   here[here],
>   lintr[lint, lint_dir],
> )
> lint(
>   filename = here("functions", "foo", "a.R"),
>   linters = box_default_linters$box_mod_fun_exists_linter
> )
> ```

This is covered in #137
